wkconnect/routes/datasets/histogram.py

```diff
@@ -30,11 +32,21 @@ async def histogram_post(
 
     layer = [i for i in dataset.to_webknossos().dataLayers if i.name == layer_name][0]
 
-    sample_positions = generate_sample_positions(2, layer.boundingBox, 32)
-    sample_positions_distinct = list(OrderedDict.fromkeys(sample_positions))
+    sample_positions = generate_sample_positions(2, layer.boundingBox, BUCKET_SIZE)
+
+    # For the WKW backend, the bucket requests need to be bucket-aligned in the target mag
```

Comment:
Maybe this could be extracted into a function?
Also, could it be skipped for backends that are not wkw?

Reply:
> Also, could it be skipped for backends that are not wkw?

We could. But the mag-detection part would still be required. And perf may be better with 32-aligned requests (because most backends have chunked storage with a 2**x ≥ 32 size). I don't see any harm in aligned the requests.